src/ManageCourses.Api/Services/Publish/Helpers/CourseHelpers.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ManageCourses.Api.Model;
 
-namespace GovUk.Education.ManageCourses.ApiClient.Helpers
+namespace GovUk.Education.ManageCourses.Api.Services.Publish.Helpers
```

Comment:
Namespace?

Reply:
Moved to GovUk.Education.ManageCourses.Api.Helpers